frontend/src/components/NodeChart.vue

```diff
@@ -213,7 +300,123 @@ const updateChartLine = () => {
+  // to its initial state.
+  let datasets = chart.data.datasets.filter(s => s.seriesType == 'swot_node_series');
+
+  // turn on all hidden swot node series datasets
+  datasets.filter(s => s.seriesType == 'swot_node_series').forEach(function(s) {
+    s.hidden = false;
+  });
+
+  // reset page components to their default state
+  setDefaults(); 
+
+  // update the chart
+  chart.data.datasets = datasets;
+  chart.update();
+  
+}
+
+async function getStatistics () {
+    let datasets = chartStore.nodeChartData.datasets
+    // upate datasets so that is just an array arrays of the datasets.data objects
+    datasets = datasets.map(dataset => {
+        return dataset.data
+    })
+
+    let stats = await fetch(`${APP_API_URL}/data/compute_node_series`, {
+        method: 'POST',
+        headers: {'Content-Type': 'application/json'},
+        body: JSON.stringify(datasets)})
+        .then(response => response.json())
+        .then(data => {
+          return data;     
+        });
+
+    // save computed statistics to a reactive variable
+    chartStatistics.value = stats;
+}
+
+function buildChartSeries (data, xkey='p_dist_out', ykey, series_label, props={}) {
+
+  let dat = {
+    label: series_label,
+    data: data,
+    parsing: {
+      yAxisKey: ykey,
+      xAxisKey: xkey,
+    },
+    fill: props.fill || false,
+    hidden: props.hidden || false,
+    showLine: props.showLine || true,
+    borderColor: props.borderColor || 'blue',
+    borderWidth: props.borderWidth || 1,
+    pointRadius: props.pointRadius || 1,
+    seriesType: props.seriesType || 'computed_series',
+  }
+
+  return dat;
+
+}
+
+const toggleSeriesStatistics = async (visible=true) => {
+  // adds and removes computed statistics from the chart
+
+  // get the data from the chart
+  let updatedDatasets = line.value.chart.data.datasets;
+
+  if (visible) {
+
+    // compute statistics
+    await getStatistics();
+
+    // build chart series for each statistic and set it in chartData.
+    // set each of these to hidden. 
+    for (let stat in chartStatistics.value) {
+
+      // skip Interquartile range because these will added later
+      if (stat != 'q0.25' && stat != 'q0.75') {
+       let series = buildChartSeries(chartStatistics.value[stat],
+                                     'p_dist_out',
+                                     props.chosenVariable.abbreviation,
+                                     stat,
+                                     {fill:false, hidden:true});
+       updatedDatasets.push(series);
+      }
+      if (stat == 'q0.25') {
+       let series = buildChartSeries(chartStatistics.value[stat],
+                                     'p_dist_out',
+                                     props.chosenVariable.abbreviation,
+                                     'IQR',
+                                     {showLine:true, borderColor:"gray", borderWidth:1, pointRadius:0});
+       updatedDatasets.push(series);
```

Comment:
the statistics are added "behind" the series plots. is that intentional? In some cases, makes it impossible to see them

Reply:
I'm not really sure how to get around this because the IQR patch would cover the other series if it was elevated above them. I'm was thinking of adding a toggle button to turn on/off node series, as well as setting "scatter" plot as the default.

Reply:
I think both of those ideas are great. Maybe we keep the "connected" plot as the initial default but if someone toggles the statistics on, then we also automatically change it from connected to "scatter"

Reply:
I'm happy to work on these remaining issues if you don't have time. Thanks for all the help on this cool new feature!